File: apollo/src/main/java/io/muun/apollo/domain/errors/FcmTokenNotAvailableError.java
package io.muun.apollo.domain.errors;


import io.muun.apollo.external.UserFacingErrorMessages;

public class FcmTokenNotAvailableError extends UserFacingError {

    public FcmTokenNotAvailableError() {
        super(UserFacingErrorMessages.INSTANCE.fcmTokenNotAvailable());
    }
}

File: android/apolloui/src/main/java/io/muun/apollo/domain/errors/fcm/FcmTokenNotAvailableError.kt
package io.muun.apollo.domain.errors.fcm


import io.muun.apollo.data.external.UserFacingErrorMessages
import io.muun.apollo.domain.errors.ErrorClassification
import io.muun.apollo.domain.errors.UserFacingError

class FcmTokenNotAvailableError : UserFacingError(
    UserFacingErrorMessages.INSTANCE.fcmTokenNotAvailable()
) {
    override val classification = ErrorClassification.UNEXPECTED
}
